recipes/pyxid/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyxid" %}
+{% set version = "1.0" %}
+{% set sha256 = "e689bce2ed8ed4bff7174a3a79f8b639ebf6356807aec7e9da481217313214ff" %}
+
+package:
+  name: pyxid
+  version: "1.0"
+
+source:
+  fn: pyxid-{{ version }}.tar.gz
+  url: https://github.com/cedrus-opensource/pyxid/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyserial >=2.5
+
+
+  run:
+    - python
+    - pyserial >=2.5
+
+test:
+  # Python imports
+  imports:
+    - pyxid
+
+about:
+  home: http://www.github.com/cedrus-opensource/pyxid/
+  license: BSD
```

Comment:
What version of BSD (e.g. BSD 3-Clause)? Also is there a license file that we can include in the package from the source?